websites/docs.temporal.io/package.json

```diff
@@ -39,10 +39,12 @@
     "@mdx-js/react": "^3.0.0",
     "chart.js": "^4.4.1",
     "clsx": "^1.1.1",
+    "docusaurus": "^1.14.7",
```

Comment:
Curious why these dependencies are being added here?

Reply:
I am not sure. This seems to be generated via the artifact creation process with yarn. There were some hiccups along the way where it wasn't building things right, so I went back to my original method.